### PR TITLE
Added support for /me

### PR DESCRIPTION
**Pull request by slavikm**
## Related Issues

Python support https://github.com/descope/etc/issues/619

## Description

Support /me request to return dict of user details

### CODE REVIEW

**Comment by guyp-descope** [comment]
@omercnet can you please check why the build failed here? what happen to Black..

File: descope/descope_client.py [hunk 1]
```diff
@@ -107,5 +107,41 @@ def logout(self, refresh_token: str) -> requests.Response:
         uri = EndpointsV1.logoutPath
         return self._auth.do_get(uri, None, None, refresh_token)
 
+    def me(self, refresh_token: str) -> dict:
```
Comment (guyp-descope) [inline descope/descope_client.py:110]: return value is requests.Response (and not dict)

File: descope/descope_client.py [hunk 2]
```diff
@@ -107,5 +107,41 @@ def logout(self, refresh_token: str) -> requests.Response:
...
+        """
+        Retrieve user details for the refresh token. The returned data includes email, name, phone,
+            list of externalIds and boolean flags for verifiedEmail, verifiedPhone.
+
+        Args:
+        refresh_token (str): The refresh token
+
+        Return value (requests.Response): returns the response from the Descope server
```
Comment (guyp-descope) [inline descope/descope_client.py:118]: maybe we should specify what is inside the response data?

File: descope/descope_client.py [hunk 3]
```diff
@@ -107,5 +107,41 @@ def logout(self, refresh_token: str) -> requests.Response:
...
+
+
     def refresh_token(self, refresh_token: str) -> dict:
+        """
+        Return a new session token for the given refresh token
+
+        Args:
+        refresh_token (str): The refresh token
+
+        Return value (requests.Response): returns the response from the Descope server
```
Comment (guyp-descope) [inline descope/descope_client.py:142]: return value here is dict (and not requests.Response)

**Review by guyp-descope — APPROVED** [review]

**Review by omercnet — APPROVED** [review]

**Pull request merged**